src/bits.rs

```diff
@@ -16,6 +16,17 @@ use core::{
     ops::{BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign, Not},
 };
 
+pub(crate) fn trailing_zeros(words: &[Word]) -> usize {
+    debug_assert!(words.len() > 0 || *words.last().unwrap() != 0);
```

Comment:
It can prevent the case where words is `[0]`. It's from your original code https://github.com/tczajka/ibig-rs/blob/4b7bee16fcb8024b2501d8a3bfc386048d89f223/src/bits.rs#L144